packages/tables/src/Columns/ImageColumn.php

```diff
@@ -153,9 +170,14 @@ public function getWidth(): ?string
         return $width;
     }
 
-    public function isRounded(): bool
```

Comment:
Please reintroduce this method & deprecate. Call `$this->isCircular()` internally. Also update docs if there are any?